client/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="style.css" />
+		<script defer src="index.js"></script>
+		<title>Google</title>
+	</head>
+	<body>
+		<header>
+			<ul>
+				<li>
+					<a class="links" href="#user"
+						><button class="signbutton" type="button">Sign in</button></a
+					>
+				</li>
+				<li>
+					<a href="#grid"
+						><img
+							class="grid"
+							src="https://cdn3.iconfinder.com/data/icons/navigation-and-settings/24/Material_icons-01-11-512.png"
+							title="Google apps"
+					/></a>
+				</li>
+				<li><a href="#images">Images</a></li>
+				<li><a href="#gmail">Gmail</a></li>
+			</ul>
+		</header>
+
+		<main>
+			<section id="logo">
+				<img id="googleLogo" src="./images/Google_2015_logo.svg.png" />
+			</section>
+			<section id="searchBar">
+				<div class="bar">
+					<form action="">
+						<input class="searchbar" type="text" title="Search" />
+					</form>
+					<a href="#">
+						<img
+							class="voice"
+							src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Google_mic.svg/716px-Google_mic.svg.png"
+							title="Search by Voice"
+					/></a>
+				</div>
+				<div class="buttons">
+					<button id='googleSearch' class="button" type="button">Google Search</button>
+					<button class="button" type="button">I'm Feeling Lucky</button>
+				</div>
+			</section>
+			<section id="break"></section>
+		</main>
+
+		<footer>
+			<p id="country">United Kingdom</p>
+
+			<ul id="info">
+				<li>Advertising</li>
+				<li>Business</li>
+				<li>How Search Works</li>
+			</ul>
+
+			<p id="carbon">Carbon neutral since 2007</p>
+
+			<ul id="otherInfo">
+				<li>Privacy</li>
+				<li>Terms</li>
+				<li>Settings</li>
+			</ul>
+		</footer>
+	</body>
+</html>
```

Comment:
Would be nice to have a favicon
Image tags have no alt attribute
Otherwise all looks good. 
Indentation is a bit erratic but it's pretty readable.